kalite/facility/api_resources.py

```diff
@@ -177,6 +178,8 @@ def generate_status(self, request, **kwargs):
             "version": version.VERSION,
             "facilities": request.session.get("facilities"),
             "simplified_login": settings.SIMPLIFIED_LOGIN,
+            "docs_exist": True if any(map(lambda x: os.path.join("sphinx-docs", "_build") in x,
```

Comment:
I believe you can remove the `if` statement and just use `any`, which returns a boolean.